build.sbt

```diff
@@ -85,34 +68,35 @@ lazy val server = (projectMatrix in file("server"))
 lazy val agent = (projectMatrix in file("agent"))
   .settings(
     commonSettings ++ Seq(
-      name := "atum-agent",
+      name := "agent",
       libraryDependencies ++= Dependencies.agentDependencies(
         if (scalaVersion.value == Versions.scala211) Versions.spark2 else Versions.spark3,
         scalaVersion.value
       ),
+      printSparkScalaVersion := {
+        val log = streams.value.log
+        val sparkVer = sparkVersionForScala(scalaVersion.value)
+        log.info(s"Building ${name.value} with Spark $sparkVer, Scala ${scalaVersion.value}")
+      },
       (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value,
-      mergeStrategy
+      jacocoReportSettings := jacocoSettings(scalaVersion.value, "atum-agent")
     ): _*
   )
-  .settings(
-    jacocoReportSettings := jacocoSettings(scalaVersion.value, "atum-agent"),
-    jacocoExcludes := jacocoProjectExcludes()
-  )
-  .sparkRow(SparkVersionAxis(Versions.spark2), scalaVersions = Seq(Versions.scala211, Versions.scala212))
-  .sparkRow(SparkVersionAxis(Versions.spark3), scalaVersions = Seq(Versions.scala212, Versions.scala213))
   .jvmPlatform(scalaVersions = Versions.clientSupportedScalaVersions)
   .dependsOn(model)
 
 lazy val model = (projectMatrix in file("model"))
   .settings(
     commonSettings ++ Seq(
-      name         := "atum-model",
+      name := "model",
       libraryDependencies ++= Dependencies.modelDependencies(scalaVersion.value),
+      printSparkScalaVersion := {
```

Comment:
shouldn't we create also a custom Task: https://www.scala-sbt.org/1.x/docs/Custom-Settings.html that will be related only to Scala? Something like `printScalaVersion` ? Because currently only the agent needs Spark. Later also Server will need it, because it will do some aggregation operations on Streaming data, but for now it doesn't need it.A

Reply:
For the record, I don't see the Server ever needing Spark. The aggregation of the partial values won't need (really shouldn't) any Spark.

Reply:
Could be, I didn't think about it that far. But the original point is still true for foreseeable future, spark is only used in Agent

Reply:
perhaps the only last thing outstanding

Reply:
Sorry, I did not see this as a task, just as a future discussion. I will create it. :)